Image processing/Image compression analyser/visualizer/diff_generator.py

```
@@ -0,0 +1,263 @@
+            for x in range(hist_width):
+                x_bin = int(x * 256 / hist_width)
+                if x_bin < 256:
+                    height = int(comp_hist[x_bin])
+                    draw.line([(x, y_offset + hist_height), 
+                             (x, y_offset + hist_height - height)], 
+                            fill=(255, 0, 0), width=1)
+            
+            # Add channel label
+            try:
+                font = ImageFont.truetype("arial.ttf", 12)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)